anaconda.spec.in

```diff
@@ -112,13 +112,15 @@ Requires: subscription-manager >= %{subscriptionmanagerver}
 # which is apparently great for containers but unhelpful for the rest of us
 Requires: cracklib-dicts
 
+%if ! 0%{?rhel}
```

Comment:
I guess it would nicer to specify the version. I know that this spec file change will not  reach el9 branch, but I think it's nice for our own understanding - of what is supported where.

Reply:
Like `%if 0%{?rhel} >= 10` ?

Reply:
I think you want > 10 but yes.

Reply:
>= 10

Reply:
Is it really necessary? We don't check for fedora anywhere else.
If I understand using of this macro correctly, `rhel` being empty means fedora, as in `%if ! 0{?rhel}` used elsewhere. And this will pass on Fedora also in case of this check (0 < 10). I thought the checks don't allow for OR so checking also for fedora additionally/explicitly could become quite complicated condition...